source/api/presentation/3.0/index.md

```diff
@@ -459,33 +459,33 @@ The value _MUST_ be a string, taken from the table below or a full URI.
 {: .api-table}
 
 ``` json-doc
-{"renderingDirection": "left-to-right"}
+{"viewingDirection": "left-to-right"}
 ```
 
-##### renderingHint
-A hint to the client as to the most appropriate method of displaying the resource. This specification defines the values specified in the table below. Others may be defined externally, and would be given as a full URI.
+##### behavior
+A set of flags that describe requested methods for the client to use when displaying the resource to the user. This specification defines the values specified in the table below. Others may be defined externally, and would be given as a URI.
```

Comment:
I don't like "methods"...

A set of flags that provide the client with details of the required user experience for the resource.

Reply:
:+1 on @tomcrane's suggestion of focusing on the UX expected, otherwise PR looks good

Reply:
Agree with this change.

Reply:
Not all behaviors are MUST, is ‘required’ too strong?   Otherwise +1

Reply:
Reading again, I'm not sure that I like "flags" either.

Reply:
"properties"?

Reply:
"hints" as an echo of their former name? They _are_ hints; the particular user experience a client viewer might construct to convey the pagedness of a digitised book (for example) is up to the imagination of viewer developers.

Reply:
Using "hints" in place of "flags" would be fine by me

Reply:
Definitely not "properties", as that sounds like json object properties. Without hint in the name, it seems strange to call them hints. Given the rename to behavior, they seem a little more ... insistent? ... than just a "hint". A request to the viewer to do something that shouldn't be ignored, rather than a hint that the experience might be better. "behaviors" ?